game/models.py

```
@@ -515,4 +515,25 @@ def save(self, *args, **kwargs):
+
+    discussion = models.ForeignKey(
+        Discussion,
+        on_delete=models.CASCADE,
+        related_name="bookmarks"
+    )
+
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        unique_together = ("user", "discussion")
+        ordering = ["-created_at"]
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)